wpical/src/main/native/cpp/WPIcal.cpp

```diff
@@ -193,7 +260,7 @@ static void DisplayGui() {
   if (ImGui::Button("Calibrate!!!")) {
     if (!selected_field_calibration_directory.empty() &&
         !selected_camera_intrinsics.empty() && !selected_field_map.empty()) {
-      download_directory_selector =
+      auto download_directory_selector =
```

Comment:
This needs to be a global and the if (download_directory_selector) needs to move to the top level, same as the other file open cases.